include/ddc/kernels/splines/matrix_sparse.hpp

```diff
@@ -41,6 +43,14 @@ auto to_gko_dense(std::shared_ptr<const gko::Executor> const& gko_exec, KokkosVi
                    view.stride_0());
 }
 
+/**
+ * @brief Return the default value of the hyperparameter cols_per_chunk for a given Kokkos::ExecutionSpace.
+ *
+ * The values are hardware-specific (but they can be overriden in the constructor of MatrixSparse). They have been tuned on the basis of ddc/benchmarks/splines.cpp results.
```

Comment:
You say that the values are hardware-specific and on which tests they have been tuned but not on which hardware.